modules/stochastic_tools/src/surrogates/PolynomialChaos.C

```diff
@@ -133,6 +57,30 @@ PolynomialChaos::evaluate(const std::vector<Real> & x) const
   return val;
 }
 
+const std::vector<std::vector<unsigned int>> &
+PolynomialChaos::getPolynomialOrders() const
+{
+  return _tuple;
+}
+
+unsigned
+PolynomialChaos::getPolynomialOrder(const unsigned int dim, const unsigned int i) const
+{
+  return _tuple[i][dim];
+}
+
+const std::vector<Real> &
+PolynomialChaos::getCoefficients() const
+{
+  return _coeff;
+}
+
+Real
+PolynomialChaos::computeMean() const
+{
+  return _coeff[0];
```

Comment:
Is it ever possible for this vector to be unsized?

Reply:
Actually I think so, if the loaded data has the variable, but it is empty. I will add a `mooseAssert` in the next relevant PR.

Reply:
That is a good catch. I will add a follow on PR that includes an assert.